styleguide/derek/templates/thought-leadership-overview/_partials/nav.ejs

```diff
@@ -0,0 +1,22 @@
+<div class="rsTl-nav-container">
+  <nav class="rsTl-nav">
+    <div class="rsTl-nav-logo">
+      <img class="rsTl-nav-img" src="imgs/derek/logos/Solve_Logo.svg" alt="solve logo" />
+    </div>
+    <div class="rsTl-nav-links">
+      <ul class="rsTl-nav-list"></ul>
+    </div>
+    <div class="rsTl-nav-rsLogo">
+      <%- partial('rs-logo') %>
```

Comment:
For whatever reason, this logo is not showing up in IE11 until you click on it. I believe it's related to this issue: 
https://stackoverflow.com/questions/34008998/svg-text-disappears-in-ie-until-i-click-on-it